src/psyclone/psyir/nodes/container.py

```diff
@@ -142,15 +145,6 @@ def name(self, new_name):
         '''
         self._name = new_name
```

Comment:
Since we're here, would you mind disabling the 'protected-access' pylint warnings in the `create` method?

Reply:
Done